bedrock/mozorg/templates/mozorg/technology.html

```diff
@@ -116,10 +129,13 @@
       <div class="content">
         <div class="inner-container">
           <h2>{{ _('Inventing a safer programming language') }}</h2>
-          <p>{{ _('Supported by Mozilla, Rust allows browsers, systems and more to run much faster and more safely.') }}</p>
-          {% block rust_cta %}
-          <a rel="external" href="https://www.youtube.com/watch?list=PLo3w8EB99pqJ74XIGe72c9hBZWz9Y16cY&v=8EPsnf_ZYU0" class="button" data-video-title="Rust and the Future of Systems Programming">{{ _('Watch the video') }}</a>
-          {% endblock %}
+          {% if l10n_has_tag('emerging_tech_links') %}
+            <p>{{ _('Sponsored by Mozilla, Rust allows browsers, systems and more to run much faster and more safely.') }}</p>
+            <a rel="external" href="https://research.mozilla.org/rust/" class="button">{{ _('Learn about Rust') }}</a>
+          {% else %}
+            <p>{{ _('Supported by Mozilla, Rust allows browsers, systems and more to run much faster and more safely.') }}</p>
```

Comment:
It's not (sponsored vs supported), and I know because I failed like you to notice the difference the first time I checked.